homeassistant/components/config/zwave.py

```diff
@@ -196,3 +197,51 @@ def get(self, request, node_id):
                                       'label': value.label,
                                       'length': len(value.data)}
         return self.json(usercodes)
+
+
+class ZWaveProtectionView(HomeAssistantView):
+    """View for the protection commandclass of a node."""
+
+    url = r"/api/zwave/protection/{node_id:\d+}"
+    name = "api:zwave:protection"
+
+    @ha.callback
+    def get(self, request, node_id):
+        """Retrieve the protection commandclass options of node."""
+        nodeid = int(node_id)
+        hass = request.app['hass']
+        network = hass.data.get(const.DATA_NETWORK)
+        node = network.nodes.get(nodeid)
+        if node is None:
+            return self.json_message('Node not found', HTTP_NOT_FOUND)
+        protection_options = {}
+        if not node.has_command_class(const.COMMAND_CLASS_PROTECTION):
+            return self.json(protection_options)
+        protections = node.get_protections()
+        protection_options = {
+            'value_id': '{0:d}'.format(list(protections.keys())[0]),
```

Comment:
I don't think we need the `.keys()`. Passing a dict into a list copy will automatically make a list out of the keys.